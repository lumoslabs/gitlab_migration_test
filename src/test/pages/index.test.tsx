import * as React from 'react'
import { render } from '@test/render-with-redux'

import { Index } from '@pages/index'


describe('Index page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Index />)
    expect(asFragment()).toMatchSnapshot()
  })
})
