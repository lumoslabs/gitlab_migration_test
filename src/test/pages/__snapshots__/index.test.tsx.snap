// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page matches snapshot 1`] = `
<DocumentFragment>
  <main>
    <div
      class="flexRowJustifyCenter_1uq28sg-o_O-fullWidth_hvhp2z-o_O-topSpace_n5r770"
    >
      <div
        class="flexAlignCenter_115lfal"
      >
        <div
          class="flexColumnAllCenter_1vexgx1-o_O-section_rgclyh"
        >
          <div
            class="flexColumnAllCenter_1vexgx1-o_O-card_1rrn0ox card"
          >
            <div
              class="flexColumnAllCenter_1vexgx1-o_O-cardBody_1858xbs card-body"
            >
              <img
                alt="Workout Icon"
                class="card-img cardImg_3mv33q"
                src="/assets/workout_icon.svg"
              />
              <div
                class="cardTitle_ena3cq card-title h5"
              >
                Daily Workout
              </div>
              <div
                class="cardSubtitle_1g4oaq4 card-subtitle h6"
              >
                Your daily workout of 3 games.
              </div>
              <div
                class="flexJustifyCenter_jfsfb5"
              >
                <button
                  class="button_1xm70sk-o_O-wideAction_4zoj1f-o_O-cta_1jbl92e"
                >
                  Start My Workout
                </button>
              </div>
            </div>
          </div>
        </div>
        <div
          class="flexRowAllCenter_1cn94ih-o_O-gamesDiv_1idp2zd container"
        >
          <div
            class="flexRowAllCenter_1cn94ih-o_O-gridCol_1f8v8h4"
          >
            <a
              href="/game/train-of-thought-nest"
            >
              <div
                class="flexRowAllCenter_1cn94ih-o_O-gridCol_1f8v8h4"
              >
                <div
                  class="flexColumnAllCenter_1vexgx1-o_O-card_2cv6tx card"
                >
                  <img
                    alt="Game icon"
                    class="card-img-top cardImg_1inpevm"
                    src="https://assets.nest.lumosity.com/frontend_assets/banner/train_of_thought_header.png"
                  />
                  <div
                    class="flexColumn_n8t8ll-o_O-cardBody_ly1891 card-body"
                  >
                    <p
                      class="pageTitle_1a3u8g5-o_O-subTitle_rlrlr4"
                    >
                      ATTENTION
                    </p>
                    <div
                      class="title_1wry8v1 card-title h5"
                    >
                      Train of Thought
                    </div>
                  </div>
                </div>
              </div>
            </a>
            <a
              href="/game/color-match-nest"
            >
              <div
                class="flexRowAllCenter_1cn94ih-o_O-gridCol_1f8v8h4"
              >
                <div
                  class="flexColumnAllCenter_1vexgx1-o_O-card_2cv6tx card"
                >
                  <img
                    alt="Game icon"
                    class="card-img-top cardImg_1inpevm"
                    src="https://assets.nest.lumosity.com/frontend_assets/banner/color_match_header.png"
                  />
                  <div
                    class="flexColumn_n8t8ll-o_O-cardBody_ly1891 card-body"
                  >
                    <p
                      class="pageTitle_1a3u8g5-o_O-subTitle_rlrlr4"
                    >
                      FLEXIBILITY
                    </p>
                    <div
                      class="title_1wry8v1 card-title h5"
                    >
                      Color Match
                    </div>
                  </div>
                </div>
              </div>
            </a>
          </div>
          <div
            class="flexRowAllCenter_1cn94ih-o_O-gridCol_1f8v8h4"
          >
            <a
              href="/game/ebb-and-flow-nest"
            >
              <div
                class="flexRowAllCenter_1cn94ih-o_O-gridCol_1f8v8h4"
              >
                <div
                  class="flexColumnAllCenter_1vexgx1-o_O-card_2cv6tx card"
                >
                  <img
                    alt="Game icon"
                    class="card-img-top cardImg_1inpevm"
                    src="https://assets.nest.lumosity.com/frontend_assets/banner/ebb_and_flow_header.png"
                  />
                  <div
                    class="flexColumn_n8t8ll-o_O-cardBody_ly1891 card-body"
                  >
                    <p
                      class="pageTitle_1a3u8g5-o_O-subTitle_rlrlr4"
                    >
                      FLEXIBILITY
                    </p>
                    <div
                      class="title_1wry8v1 card-title h5"
                    >
                      Ebb and Flow
                    </div>
                  </div>
                </div>
              </div>
            </a>
            <a
              href="/game/raindrops-nest"
            >
              <div
                class="flexRowAllCenter_1cn94ih-o_O-gridCol_1f8v8h4"
              >
                <div
                  class="flexColumnAllCenter_1vexgx1-o_O-card_2cv6tx card"
                >
                  <img
                    alt="Game icon"
                    class="card-img-top cardImg_1inpevm"
                    src="https://assets.nest.lumosity.com/frontend_assets/banner/raindrops_header.png"
                  />
                  <div
                    class="flexColumn_n8t8ll-o_O-cardBody_ly1891 card-body"
                  >
                    <p
                      class="pageTitle_1a3u8g5-o_O-subTitle_rlrlr4"
                    >
                      MATH
                    </p>
                    <div
                      class="title_1wry8v1 card-title h5"
                    >
                      Raindrops
                    </div>
                  </div>
                </div>
              </div>
            </a>
          </div>
          <div
            class="flexRowAllCenter_1cn94ih-o_O-gridCol_1f8v8h4"
          >
            <a
              href="/game/word-snatchers-nest"
            >
              <div
                class="flexRowAllCenter_1cn94ih-o_O-gridCol_1f8v8h4"
              >
                <div
                  class="flexColumnAllCenter_1vexgx1-o_O-card_2cv6tx card"
                >
                  <img
                    alt="Game icon"
                    class="card-img-top cardImg_1inpevm"
                    src="https://assets.nest.lumosity.com/frontend_assets/banner/word_snatchers_header.png"
                  />
                  <div
                    class="flexColumn_n8t8ll-o_O-cardBody_ly1891 card-body"
                  >
                    <p
                      class="pageTitle_1a3u8g5-o_O-subTitle_rlrlr4"
                    >
                      LANGUAGE
                    </p>
                    <div
                      class="title_1wry8v1 card-title h5"
                    >
                      Word Snatchers
                    </div>
                  </div>
                </div>
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="flexRow_1r5z645-o_O-footer_smmve6"
    >
      <div
        class="flexRow_1r5z645-o_O-clickable_162zso0"
      >
        <img
          alt="Avatar"
          class="avatarImg_yvir6c"
          src="/assets/guest_avatar.svg"
        />
        <p
          class="nameText_5s9o36"
        >
          Guest
        </p>
      </div>
    </div>
  </main>
</DocumentFragment>
`;
